src/main/java/http/Router.java

```diff
@@ -0,0 +1,33 @@
+package http;
+
+import http.routes.*;
+
+public class Router {
+
+    public static String getResponse(String method, String path, String body) {
+        switch (path) {
```

Comment:
Note: Look into implementation with hashmaps as well - not a blocker

Reply:
Used the new `Route` interface to implement a `HashMap` instead of the `switch` statement